src/ui/component/transactionList/view.jsx

```diff
@@ -42,10 +43,8 @@ class TransactionList extends React.PureComponent<Props> {
   }
 
   isRevokeable(txid: string, nout: number) {
-    const { myClaims } = this.props;
-    // a claim/support/update is revokable if it
-    // is in my claim list(claim_list_mine)
-    return myClaims.has(`${txid}:${nout}`);
+    const { mySupports, myClaims } = this.props;
+    return !!mySupports[txid] || myClaims.has(`${txid}:${nout}`);
```

Comment:
I believe you can have multiple supports in a single transaction. Should supports also be indexed by outpoint rather than txid?

Reply:
Good point. Yeah, we probably should use it. Is this a blocker? RC testing has been using it. I can update after the release